jellyfin_kodi/objects/kodi/kodi.py

```diff
@@ -21,6 +22,14 @@ class Kodi(object):
     def __init__(self):
         self.artwork = artwork.Artwork(self.cursor)
 
+        try:
+            self.cursor.execute(QU.get_all_people)
+        except:
+            # Failed to load the table. Has the table been created?
+            self._people_cache = {}
+        else:
+            self._people_cache = dict(self.cursor.fetchall())
```

Comment:
Wouldn't it make more sense to move this up above the `except`?  I've never found a need for the else block over just putting the desired actions in the try block, so I'm curious about the choice to include it here.

Reply:
I believe it is a good pattern to use the else block of a try/catch.
It helps communicate to a reader which calls the author intended to catch from and which calls they did not anticipate catching from. It also prevents potential accidental catches of exceptions from calls you may want to propagate or learn about.
I think I read somewhere to keep the try blocks as small as possible but no smaller; Ive taken that to heart as I have (albeit small) justifications for using this pattern but not seen evidence against using it.

Reply:
Doing some quick research, it does appear that you're correct and using the else block is considered more "proper".  I've just never seen it done this way before.